apps/patient/src/views/Pharmacy.tsx

```diff
@@ -99,31 +98,43 @@ export const Pharmacy = () => {
     lat: number | undefined
     lng: number | undefined
   }) => {
-    if (loc && loc !== location) {
-      reset()
-    }
-    if (lat && lng && loc) {
-      setLocation(loc)
-      setLatitude(lat)
-      setLongitude(lng)
-
-      const inAustinTX = /Austin.*(?:TX|Texas)/.test(loc)
-      const isMoPed = order?.organization?.id === process.env.REACT_APP_MODERN_PEDIATRICS_ORG_ID
-      if (inAustinTX && isMoPed) {
-        setEnableCourier(true)
+    if (loc && loc !== location) reset()
+    if (lat && lng && loc) setLocation(loc)
+    setLocationModalOpen(false)
+  }
+
+  const geocode = async (address: string) => {
+    const data = await geocoder.geocode({ address })
+    if (data?.results) {
+      return {
+        loc: data.results[0].formatted_address,
+        lat: data.results[0].geometry.location.lat(),
+        lng: data.results[0].geometry.location.lng()
       }
     }
-    setLocationModalOpen(false)
   }
 
   const fetchPharmacies = async () => {
     setLoadingMore(true)
 
+    const searchingInAustinTX = /Austin.*(?:TX|Texas)/.test(location)
+    const patientAddressInAustinTX = true
+    const isMoPed = order?.organization?.id === process.env.REACT_APP_MODERN_PEDIATRICS_ORG_ID
+    if (searchingInAustinTX && patientAddressInAustinTX && isMoPed) {
```

Comment:
`patientAddressInAustinTX` looks like it will always be true

Reply:
ah! lol yeah was using that for testing that at the end

Reply:
fixed